edgeapplications.yaml

```diff
@@ -502,11 +512,21 @@ components:
         delivery_protocol:
           type: string
         http_port:
-          format: int64
-          type: integer
+          oneOf:
+            - type: integer
```

Comment:
From what I talked to @trgalho, the api will return integers and arrays of integers and because of that we should use an interface to receive generic values